flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/debezium/reader/BinlogSplitReaderTest.java

```diff
@@ -74,7 +74,7 @@
     @Test
     public void testReadSingleBinlogSplit() throws Exception {
         customerDatabase.createAndInitialize();
-        MySqlSourceConfig sourceConfig = getConfig(new String[] {"customers"});
+        MySqlSourceConfig sourceConfig = getConfig(new String[] {"customers_2"});
```

Comment:
I don't see why we need to update this test.

Reply:
The existing tests need to update after using the dynamic chunk size assignment,  the existing table `customers` is not evenly distributed and the row number is less than its dynamic chunk size